jolly-jellyfish/src/page_maker/templates/registration/login.html

```diff
@@ -0,0 +1,24 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+    {% if form.errors %}
```

Comment:
```suggestion
{% block content %}
    <h2>Login</h2>
    {% if form.errors %}
```
As above, styling concerns.